docs/getting-started.md

```diff
@@ -31,6 +33,40 @@ npm update -g serverless
 
 If you don’t already have [Node 6](https://nodejs.org/en/download/package-manager/) or higher on your machine, you’ll need to do that first.
 
+## Installing as a standalone binary
```

Comment:
Should we advertise this as the "standard" way (e.g. move this to the top)? I guess this is a product question.

Reply:
I think, even bigger changes to docs will added, but I'd leave it to product team after it's merged and published.

In this PR I just wanted to expose that alternative method exists

Reply:
Sounds like a good plan :+1: